generics/middleware/authenticator.js

```diff
@@ -44,7 +44,7 @@ var respUtil = function (resp) {
  * @returns {Promise<Object>} Returns a promise that resolves to validation result object.
  */
 var validateOrgsPassedInHeader = async function(orgsFromHeader,tenantId){
-  let tenantInfo = await userService.fetchDefaultOrgDetails(tenantId);
+  let tenantInfo = await userService.getOrgDetails(tenantId);
```

Comment:
@praveenKDass make sure function name changes applied in all places in the code base